ipn/ipnstate/ipnstate.go

```diff
@@ -111,12 +112,12 @@ type PeerStatus struct {
 
 	// Tags are the list of ACL tags applied to this node.
 	// See tailscale.com/tailcfg#Node.Tags for more information.
-	Tags []string `json:",omitempty"`
+	Tags views.StringSlice `json:",omitempty"`
```

Comment:
does omitempty work on a view? (add a little test?)

Reply:
Apparently not, the only way around this would be to make this a pointer which is gross.